MainWindow.py

```
@@ -1,695 +1,747 @@
+					message:str = "是否前往发布页查看是否存在更新？"
+					url:str = "https://github.com/Ginsakura/MUSYNCSave/releases/latest"
+					self.root.after(0, lambda _:(webbrowser.open(url) if messagebox.askyesno(messageHead, message) else _ ))
+					return
+				else:
+					self.logger.error(resJson)
+			else: # 取最新tag版本号
+				# 取最新版本所在索引（区分rc、pre）
+				selectedTag:dict = resJson[0]
+				if (not updateChannel):
+					for ids in len(resJson):
+						if (resJson[ids]["prerelease"] == False):
```

Ruff (0.14.8)

Avoid equality comparisons to `False`; use `not resJson[ids]["prerelease"]:` for false checks

Replace with `not resJson[ids]["prerelease"]`

(E712)